pkg/plugins/resources/k8s/store.go

```diff
@@ -93,6 +94,23 @@ func (s *KubernetesStore) Delete(ctx context.Context, r core_model.Resource, fs
 	}
 	return nil
 }
+func (s *KubernetesStore) DeleteMany(ctx context.Context, fs ...store.DeleteManyOptionsFunc) error {
+	opts := store.NewDeleteManyOptions(fs...)
+
+	for _, list := range s.TypeRegistry.RegisteredLists() {
+		if err := s.Client.List(ctx, list); err != nil {
+			return errors.Wrapf(err, "failed to list resources of type %s", list.GetObjectKind().GroupVersionKind().String())
+		}
+		for _, obj := range list.GetItems() {
+			if opts.Mesh == "" || opts.Mesh == obj.GetMesh() {
```

Comment:
Support for `opts.Mesh == ""` is too dangerous (I don't think somebody will ever need this feature)